qa/common/gen_qa_model_repository

```diff
@@ -64,29 +64,29 @@ TENSORRT_IMAGE=${TENSORRT_IMAGE:=nvcr.io/nvidia/tensorrt:$TRITON_VERSION-py3}
 CUDA_DEVICE=0
 
 ###
-HOST_SRCDIR=/tmp/gen_srcdir
-HOST_DESTDIR=/tmp/$TRITON_VERSION/qa_model_repository
-HOST_VARDESTDIR=/tmp/$TRITON_VERSION/qa_variable_model_repository
-HOST_IDENTITYDESTDIR=/tmp/$TRITON_VERSION/qa_identity_model_repository
-HOST_SIGDEFDESTDIR=/tmp/$TRITON_VERSION/qa_tf_tag_sigdef_repository
-HOST_TFPARAMETERSDESTDIR=/tmp/$TRITON_VERSION/qa_tf_parameters_repository
-HOST_IDENTITYBIGDESTDIR=/tmp/$TRITON_VERSION/qa_identity_big_model_repository
-HOST_SHAPEDESTDIR=/tmp/$TRITON_VERSION/qa_shapetensor_model_repository
-HOST_RESHAPEDESTDIR=/tmp/$TRITON_VERSION/qa_reshape_model_repository
-HOST_SEQDESTDIR=/tmp/$TRITON_VERSION/qa_sequence_model_repository
-HOST_DYNASEQDESTDIR=/tmp/$TRITON_VERSION/qa_dyna_sequence_model_repository
-HOST_DYNASEQIMPLICITDESTDIR=/tmp/$TRITON_VERSION/qa_dyna_sequence_implicit_model_repository
-HOST_VARSEQDESTDIR=/tmp/$TRITON_VERSION/qa_variable_sequence_model_repository
-HOST_ENSEMBLEDESTDIR=/tmp/$TRITON_VERSION/qa_ensemble_model_repository
-HOST_NOSHAPEDESTDIR=/tmp/$TRITON_VERSION/qa_noshape_model_repository
-HOST_PLGDESTDIR=/tmp/$TRITON_VERSION/qa_trt_plugin_model_repository
-HOST_RAGGEDDESTDIR=/tmp/$TRITON_VERSION/qa_ragged_model_repository
-HOST_FORMATDESTDIR=/tmp/$TRITON_VERSION/qa_trt_format_model_repository
-HOST_IMPLICITSEQDESTDIR=/tmp/$TRITON_VERSION/qa_sequence_implicit_model_repository
-HOST_VARIMPLICITSEQDESTDIR=/tmp/$TRITON_VERSION/qa_variable_sequence_implicit_model_repository
-HOST_INITIALSTATEIMPLICITSEQDESTDIR=/tmp/$TRITON_VERSION/qa_sequence_initial_state_implicit_model_repository
-HOST_VARINITIALSTATEIMPLICITSEQDESTDIR=/tmp/$TRITON_VERSION/qa_variable_sequence_initial_state_implicit_model_repository
-HOST_TORCHTRTDESTDIR=/tmp/$TRITON_VERSION/torchtrt_model_store
+HOST_BUILD_DIR=${HOST_BUILD_DIR:=/tmp}
+HOST_SRCDIR=$HOST_BUILD_DIR/gen_srcdir
+HOST_DESTDIR=$HOST_BUILD_DIR/$TRITON_VERSION/qa_model_repository
+HOST_VARDESTDIR=$HOST_BUILD_DIR/$TRITON_VERSION/qa_variable_model_repository
+HOST_IDENTITYDESTDIR=$HOST_BUILD_DIR/$TRITON_VERSION/qa_identity_model_repository
+HOST_SIGDEFDESTDIR=$HOST_BUILD_DIR/$TRITON_VERSION/qa_tf_tag_sigdef_repository
```

Comment:
I think you have missed `qa_tf_parameters_repository`.

Reply:
I have restored it